storage/innobase/xtrabackup/test/t/xb_decompress_file_validation.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+. inc/common.sh
+
+start_server
+mysql -e "create table t1(i int)" test
+
+vlog "case#1 backup with qpress and compression"
+
```

Comment:
I assume the fix path is common to other compression libraries, Can you please check with zstd?

Reply:
yes, for 8.0. the test is with zstd . For 2.4 It is with qpress